texar/torch/data/data/multi_aligned_data_test.py

```diff
@@ -203,6 +203,29 @@ def test_length_filter(self):
              "length_filter_mode": "truncate"})
         self._run_and_test(hparams, discard_index=0)
 
+    def test_scalar_types(self):
```

Comment:
Might as well test that we correctly reject non-supported types.

Reply:
Added in 0758eb5. Added tests in `scalar_data_test.py` as well